meraki_appliance.tf

```
@@ -132,6 +132,11 @@ locals {
               policy = try(rule.policy, local.defaults.meraki.domains.organizations.networks.appliance.firewall.l7_firewall_rules.policy, null)
               type   = try(rule.type, local.defaults.meraki.domains.organizations.networks.appliance.firewall.l7_firewall_rules.type, null)
               value  = try(rule.value, local.defaults.meraki.domains.organizations.networks.appliance.firewall.l7_firewall_rules.value, null)
+              value_countries = try(length(rule.value_countries) == 0, true) ? null : [
+                for c in rule.value_countries : {
+                  country = c
+                }
+              ]
```

The other lists of scalar values have been treated as scalars so far; so I would do the same for consistency:
```suggestion
              value_countries = try(rule.value_countries, local.defaults.meraki.domains.organizations.networks.appliance.firewall.l7_firewall_rules.value_countries, null)
```
We do lose the conversion from `[]` to `null`, but I don't know if we really need it that much.

---

Ah, I didn't notice you were converting it to a list of maps. But yes, it is a list of strings: https://github.com/CiscoDevNet/terraform-provider-meraki/blob/main/gen/definitions/appliance_l7_firewall_rules.yaml#L41